gradle/libs.versions.toml

```diff
@@ -97,6 +95,15 @@ timber = { module = "com.jakewharton.timber:timber", version.ref = "timber" }
 auto-service = { module = "com.google.auto.service:auto-service", version.ref = "auto-service" }
 
 [plugins]
+android-application = { id = "com.android.application", version.ref = "android-gradle-plugin" }
+android-library = { id = "com.android.library", version.ref = "android-gradle-plugin" }
+android-lint = { id = "com.android.lint", version.ref = "android-gradle-plugin" }
+google-mobile-services = { id = "com.google.gms.google-services", version.ref = "google-services" }
+firebase-crashlytics = { id = "com.google.firebase.crashlytics", version = "2.9.9" }
```

Comment:
we probably want to move the version up so that all versions are under a single location?